modules/gitops-release-notes-1-17-0.adoc

```diff
@@ -0,0 +1,154 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-17-0.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-17-0_{context}"]
+= Release notes for {gitops-title} 1.17.0
+
+{gitops-title} 1.17.0 is available on {OCP} 4.12 and later versions.
+
+[id="errata-updates-1-17-0_{context}"]
+== Errata updates
+
+[id="RHEA-XXXX:NNNN-gitops-1-17-0-security-update-advisory_{context}"]
+=== RHEA-XXXX:NNNN - {gitops-title} 1.17.0 security update advisory
+
+Issued: 2025-08-06
+
+The list of security fixes that are included in this release are documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHEA-XXXX:NNNN[RHEA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="new-features-1-17-0_{context}"]
+== New features
+
+[id="gitops-1-17-0-argo-rollous-integration-ocp-dev-console-topology_{context}"]
+=== Argo Rollouts integration in OpenShift Developer Console Topology view
+
+* With this update, a new Developer Preview feature adds support for displaying Argo Rollout resources in the OpenShift Developer Console's Topology view. Rollout instances now appear as individual nodes, improving visibility into progressive delivery workflows. To use this feature, install the GitOps 1.17.0 Operator on OpenShift Container Platform version 4.19.0 or later. link:https://issues.redhat.com/browse/GITOPS-5870[GITOPS-5870]
```

Comment:
I still have to verify, but a required fix for a Console issue is in 4.19.6.  So, please put 4.19.6 or later here.

Also, what about the contents I added to the release note section of epic 5870 (Link above)?  I thought that was needed to warn users about the implications of this feature being a Developer Preview item.

Reply:
Hey @keithchong, let me get back to you on that soon.